ajax/libs/vis-timeline/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "vis-timeline",
+  "npmName": "vis-timeline",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*"
```

Comment:
Is `types/index.d.ts` not needed?

Reply:
I am not exactly sure. I do not think it is needed (at least for me). I added it back just in case.